ext/dcos-installer/dcos_installer/cli.py

```diff
@@ -50,7 +50,15 @@ def print_usage():
 Install Mesosophere's Data Center Operating System
```

Comment:
I think we can, we removed it initially to support overriding the PORT env from the docker wrapper (at least that's how I recall the decision to write this in manually, but don't quote me on it).